connector/connector.py

```diff
@@ -443,10 +443,11 @@ def unwrap_binding(self, binding_id, browse=False):
         else:
             binding = self.model.browse(binding_id)
 
-        openerp_record = getattr(binding, self._openerp_field)
+        # XXX: do we expect to have a m2o field here?
```

Comment:
`self._openerp_field` should be the m2o fields towards the "normal" record (the m2o used in the `_inherits`), does it answer to you question?